CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### 🐞 Fixed
+- Fix an issue where member role sent from backend was not recognized by the SDK [#1279](https://github.com/GetStream/stream-chat-swift/issues/1279)
```

Comment:
if PR fixes an issue should we reference the PR or the issue from the CHANGELOG? 🤔
cc @b-onc

Reply:
doesn't matter that much, but I'd prefer
PR refers to issue (fixes #xxxx)
changelog refers to PR

Reply:
Makes sense, I'll make this change 👍